### PR TITLE
[15.0][MIG] account_move_line_mrp_info

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

File: account_move_line_mrp_info/tests/test_mrp_journal_items.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
+from odoo.tests import Form
+
+from odoo.addons.mrp.tests.common import TestMrpCommon
+
+
+class TestMrpOrder(TestMrpCommon):
+    def setUp(self):
+        super(TestMrpOrder, self).setUp()
```
Comment (MiquelRForgeFlow) [inline account_move_line_mrp_info/tests/test_mrp_journal_items.py:8]:
```suggestion
    @classmethod
    def setUpClass(cls):
        super(TestMrpOrder, cls).setUpClass()
```

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review + code

**Pull request merged**